Nest Troubleshooting info for "No Access to Partner Information" Error Update nest.markdown

genestealer opened the pull request:
Nest Troubleshooting infp for "No Access to Partner Information" Error Update nest.markdown

## Proposed change
This pull request adds new troubleshooting information to the documentation to help users resolve the "No access to partner information could not be retrieved" error message encountered during the setup wizard. The added content emphasizes the importance of migrating a Nest Account to a Google Account using the Google Nest App and provides guidance for situations where a home has multiple members. This information aims to improve the user experience and assist them in successfully linking to Home Assistant.



## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
Example of error message:
![image](https://github.com/home-assistant/home-assistant.io/assets/5207490/12f325ac-2d40-4125-95c6-c1dccc12e8ba)


## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

genestealer committed:
Added requirement to migrate next account to google account "No Access To Partner” error - Update nest.markdown

Added requirement to migrate next account to google account "No Access To Partner” error - Update nest.markdown

genestealer committed:
Update nest.markdown

allenporter commented on source/_integrations/nest.markdown:
```diff
@@ -466,6 +466,8 @@ Once you have completed the above steps, you can continue through the flow to re
 
 ## Troubleshooting
 
+- *No access to partner information* "Infomation could not be retrieved" message during the setup wizard, please ensure that you have successfully migrated your Nest Account to a Google Account using the Google Nest App. Additionally, if your home has multiple members, please note that the individual who initially set up the home must complete the migration of their Nest Account to a Google Accou ...
```
I appreciate adding the documentation, thank you.

My impression is that this error message means that the account is not able to access the Home, and there are multiple reasons for this -- where the issue you'e cited here is probably one of the most common. I've made a slight adjustment to give a little more context.


```suggestion
- *No access to partner information* "Infomation could not be retrieved" message during the setup wizard means that the Google Account used is not able to access the Home.  Please ensure that you have successfully migrated your Nest Account to a Google Account using the Google Nest App. Additionally, if your home has multiple members, please note that the individual who initially set up the home must complete the migration of their Nest Account to a Google Account before you can establish a connection with Home Assistant.
```

We can also link to the nest docs here as well.
https://developers.google.com/nest/device-access/reference/errors/pcm#no_access_to_partner if that would be helpful.

genestealer replied:
Sorry added a typo correction before seeing your suggestion. Will incorporate your edit now.

allenporter reviewed (changes requested):
Thank you for improving the documentation!  👍🏼

genestealer committed:
Typo Information fixed

genestealer committed:
Added edit as per feedback

As per https://github.com/home-assistant/home-assistant.io/pull/29085#discussion_r1342151285

genestealer commented:
@allenporter your suggested change has been incorporated, along with typo correction to my contribution.

allenporter commented on source/_integrations/nest.markdown:
```diff
@@ -466,6 +466,8 @@ Once you have completed the above steps, you can continue through the flow to re
 
 ## Troubleshooting
 
+- *No access to partner information* "Information could not be retrieved" error message during the setup wizard means that the Google Account used is not able to access the Home.  Please ensure that you have successfully migrated your Nest Account to a Google Account using the Google Nest App. Additionally, if your home has multiple members, please note that the individual who initially set up th ...
```
```suggestion
- *No access to partner information* "Information could not be retrieved" error message during the setup wizard means that the Google Account used is not able to access the Home. Please ensure that you have successfully migrated your Nest Account to a Google Account using the Google Nest App. Additionally, if your home has multiple members, please note that the individual who initially set up the home must complete the migration of their Nest Account to a Google Account before you can establish a connection with Home Assistant.
```

allenporter replied:
I added an extra space.

allenporter committed:
Update source/_integrations/nest.markdown
